omero/developers/Server/LightAdmins.txt

```diff
@@ -0,0 +1,244 @@
+The server's view of administrator restrictions
+===============================================
+
+OMERO 5.4 introduced the concept of a restricted administrator. These
+are generally more powerful than normal OMERO users and group owners but
+they do not have all the powers of full administrators such as ``root``
+because they are restricted in specific ways by their
+``Experimenter.config`` property. For that property, a name of
+``AdminPrivilege:Dance`` (just an example!) with a value of ``false``
+indicates that the user's ability to dance is not that of a full
```

Comment:
Felt the same way, somehow the statement doesn't drill down the point in a simple way (assuming that the example was picked with such intention).

May be something within the list of available options, like chggrp or chown, itself might drive the point better and keep the context within omero?